src/averageAges.js

```diff
@@ -38,6 +49,18 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const sortedPeople = withChildren
+    ? people.filter((person) =>
+      person.sex === 'f'
```

Comment:
There are a few places where the same logic for checking person's sex is reused. Consider creating the function isMan or isWoman for this purpose